website/static/img/homepage/cross-platform.svg

```diff
@@ -1 +1 @@
family="Helvetica" font-size="18"><tspan x="28.986328" y="33">Body.js</tspan></text></g><g transform="translate(0 325)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="37.986328" y="33">Image</tspan></text></g><g transform="translate(151 325)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="45.496582" y="33">Text</tspan></text></g><g transform="translate(372 191)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="45.496582" y="33">Text</tspan></text></g><g transform="translate(372 260)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="45.496582" y="33">Text</tspan></text></g><g transform="translate(372 329)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="45.496582" y="33">Text</tspan></text></g><g transform="translate(372 397)"><rect height="52" rx="8" stroke="#999" stroke-width="3" width="123" x="1.5" y="1.5"/><text fill="#3d3d3d" font-family="Helvetica" font-size="18"><tspan x="45.496582" y="33">Text</tspan></text></g><path d="m201 304h241v6h-241z" fill="#999" transform="matrix(0 1 1 0 14.5 -14.5)"/><g transform="matrix(1 0 0 -1 323 227)"><path d="m0 8.5h1.49809915 27.50190085" stroke="#999" stroke-width="6"/><path d="m40.2835353 9.57993401c.9324575-.59631093.9356445-1.56108391 0-2.15943291l-9.914219-6.34018906c-1.860822-1.19000433-3.3693163-.37037919-3.3693163 1.83629677v11.16721749c0 2.204163 1.5041128 3.0291031 3.3693163 1.8362968z" fill="#999"/></g><g transform="matrix(1 0 0 -1 323 365)"><path d="m0 8.5h1.49809915 27.50190085" stroke="#999" stroke-width="6"/><path d="m40.2835353 9.57993401c.9324575-.59631093.9356445-1.56108391 0-2.15943291l-9.914219-6.34018906c-1.860822-1.19000433-3.3693163-.37037919-3.3693163 1.83629677v11.16721749c0 2.204163 1.5041128 3.0291031 3.3693163 1.8362968z" fill="#999"/></g><g transform="matrix(1 0 0 -1 323 296)"><path d="m0 8.5h1.49809915 27.50190085" stroke="#999" stroke-width="6"/><path d="m40.2835353 9.57993401c.9324575-.59631093.9356445-1.56108391 0-2.15943291l-9.914219-6.34018906c-1.860822-1.19000433-3.3693163-.37037919-3.3693163 1.83629677v11.16721749c0 2.204163 1.5041128 3.0291031 3.3693163 1.8362968z" fill="#999"/></g><g transform="matrix(1 0 0 -1 323 432)"><path d="m0 8.5h1.49809915 27.50190085" stroke="#999" stroke-width="6"/><path d="m40.2835353 9.57993401c.9324575-.59631093.9356445-1.56108391 0-2.15943291l-9.914219-6.34018906c-1.860822-1.19000433-3.3693163-.37037919-3.3693163 1.83629677v11.16721749c0 2.204163 1.5041128 3.0291031 3.3693163 1.8362968z" fill="#999"/></g></g></svg>
```

Comment:
What is this change for?

Reply:
Font change in SVG graph on the front page:
<img width="999" alt="Screenshot 2022-01-26 at 19 32 39" src="https://user-images.githubusercontent.com/719641/151224996-175424e3-ebf7-4b49-9e79-0100edb8cd4b.png">

Reply:
Wow you really made sure all fonts were changed!